lib/cuke_linter/helpers/dialect_helper.rb

```diff
@@ -0,0 +1,25 @@
+module CukeLinter
+  module DialectHelper
+
+    def self.set_dialect(dialect)
+      @dialect = dialect
+    end
+
+    def self.given_keyword
+      @dialect['given'][1].strip
+    end
+
+    def self.when_keyword
+      @dialect['when'][1].strip
+    end
+
+    def self.then_keyword
+      @dialect['then'][1].strip
+    end
+
+    def self.get_model_dialect(model)
+      set_dialect(Gherkin::DIALECTS[model.parent_model.parsing_data[:language]])
+      self
```

Comment:
On the one hand, bonus points for enabling method chaining. On the other hand, I'd expect code like `DialectHelper.get_model_dialect(model).when_keyword` to work because the `get_dialect` method returned some kind of dialect object, not because `DialectHelper` was actually receiving both messages and it secretly relied on a 'side effect' contained in a 'getter' method.